src/ipbb/generators/vivadoproject.py

```diff
@@ -180,7 +180,7 @@ def write(self, aOutput, aSettings, aComponentPaths, aCommandList, aLibs):
             # we apply a work-around.
             os_info = read_os_release()
             need_workaround = os_info \
-                and (os_info['PLATFORM_ID'] == 'platform:el8')
+                and (os_info.get('REDHAT_SUPPORT_PRODUCT_VERSION', None) == '8')
```

Comment:
When I tested this comparison in an Alma 8 container (specifically `almalinux:8.9-20231124`), I actually got false, since `REDHAT_SUPPORT_PRODUCT_VERSION` is `'8.9'` rather than `'8'`. Given that, I'm wondering if we need to just compare the first character from this string (or use a regex). Here I'm assuming that `need_workaround` should be True on all Alma 8 minor versions (I can't recall whether the sendline bug affect all RHEL8 minors and derivatives)